packages/viewer/tests/cypress/cypress.d.ts

```diff
@@ -4,34 +4,11 @@ import { mount } from 'cypress/vue'
 import type { ExternalWMSLayer, ExternalWMTSLayer } from '@swissgeo/layers'
 
 import type { GoToViewOptions, PartialLayer } from './support/commands'
+import type { MockLayer } from '@/../tests/cypress/tests-e2e/utils'
```

Comment:
I don't really get why you moved it out of here, you should have been able to import and use it in the other file, no?